plugins/auto-refetch/src/index.ts

```diff
@@ -0,0 +1,78 @@
+import type { PiniaColadaPlugin } from '@pinia/colada'
+
+export interface PiniaColadaAutoRefetchOptions {
+  /**
+   * Whether to enable auto refresh by default.
+   * @default true
+   */
+  enabled?: boolean
+}
+
+/**
+ * Plugin that automatically refreshes queries when they become stale
+ */
+export function PiniaColadaAutoRefetch(
+  options: PiniaColadaAutoRefetchOptions = {},
+): PiniaColadaPlugin {
+  const { enabled = true } = options
```

Comment:
default to `false` is probably better, no?

Reply:
My initial thinking was if someone is adding this plugin their intention would be to enable auto fetch. But I now think false is better considering it might be a large project and the plugin might be added to use it on some queries. Changing it now.